lib/Models/OgrCatalogItem.js

```diff
@@ -118,6 +118,14 @@ We recommend you upgrade to the latest version of <a href="http://www.google.com
 <a href="http://www.microsoft.com/ie" target="_blank">Microsoft Internet Explorer</a>.'
         });
     }
+    if (this.terria.configParameters.conversionServiceBaseUrl === false) {
+        // Don't allow conversion service. Duplicated in createCatalogItemFromFileOrUrl.js
+        throw new TerriaError({
+            title: 'Unsupported file type',
+            message: 'This file format is not supported by ' + this.terria.appName + '. Supported file formats include: ' +
+            '<ul><li>.geojson</li><li>.kml, .kmz</li><li>.csv (in <a href="https://github.com/NICTA/nationalmap/wiki/csv-geo-au">csv-geo-au format</a>)</li></ul>'
```

Comment:
I'm guessing you copied this text and URL from somewhere, but it should be https://github.com/TerriaJS/nationalmap/wiki/csv-geo-au instead of that NICTA one.